src/note.cpp

```diff
@@ -350,23 +352,38 @@ std::u32string Note::GetTabFretString(data_NOTATIONTYPE notationType, bool &over
         //
         // However, some glyphs are missing:
         //
-        //   Digit 1 with an oblique stroke for the open 6th course.
+        //   Digit 1 with a strike through for the open 6th course.
+        //   Digit 1 with two strike throughs for the open 7th course.
+        //   Digit 1 with three strike throughs for the open 8th course.
         //   "et" for 2nd course 5th fret.
         //   "con" for 1st course 5th fret.
         //   Gothic font digits 1-5 for the open courses <= 5.
         //   Second lowercase alphabet with an overline used for courses <= 5 frets 6 to 10.
         //
         // To overcome these omissions I've substituted missing glyphs from other
-        // parts of the SMuFL collection.  Overlines are drawn separately.
+        // parts of the SMuFL collection.  Overlines and strike throughs are drawn separately.
 
-        if (course == 6 && fret >= 0 && fret <= 13) {
-            if (fret == 0) {
-                fretStr = SMUFL_E595_ornamentLeftVerticalStrokeWithCross; // substitute for 1 with oblique stroke
-            }
-            else {
+        if (course >= 6 && fret >= 0 && fret <= 13) {
+            // TODO more GLT subtypes needed, is this subtype too specific?
+            if (notationSubtype == "judenkunig_1523") {
```

Comment:
Looking again at this PR, this is also problematic. We should not have in Verovio hardcoded values like that. For the glyphs missing I would suggest using a symbol table. This is already supported by Verovio (see the test suite examples for symbols). Pinging @musicog.

Reply:
As decided at the MEI Development Workshop at mdw, we will be defaulting to Newsidler unless an alternate symbol is defined (via `@glyphnum` or `@altsym` on `<note>`, or via a `<symbolTable>`)

Reply:
Just for reference: https://github.com/w3c/smufl/issues/274

Reply:
Does this mean that @notationsubtype is no longer to be used for the >= 6th course variants?   I.e. there are no "predefined variant sets"?

If so I can remove the @notationsubtype code and just keep the Newsidler variant.   This pull request would then  just implement the vertical positioning  with @loc and @valign and fix Issue https://github.com/rism-digital/verovio/issues/3473.   Would that be sufficient to have this pull request accepted?  Or are more changes required?

I would like to deal with implementing @glyphnum and @altsym etc in a separate pull request - first I need to study how to do this.

Thanks

Reply:
@paul-bayleaf yes please remove the `@notationsubtype`-dependent code and I can then merge the PR.

The `@valign` on `staffDef`will be renamed at some point, but the code can easily be adjusted once the renaming is done. Conceptually nothing will change so I am OK to merge it.